tests/onnxruntime/test_modeling_ort.py

```diff
@@ -0,0 +1,467 @@
+import os
```

Comment:
I didn't see any yaml file defining a workflow to handle the tests from test_modeling_ort.py ? It seems that [test_onnxruntime.yml](https://github.com/huggingface/optimum/blob/add-ort-inference/.github/workflows/test_onnxruntime.yml)  is only running the tests from test_onnxruntime.py and not all the ones from the onnxruntime directory. The tests from test_modeling_base.py would be covered by [test_optimum_common.yml](https://github.com/huggingface/optimum/blob/add-ort-inference/.github/workflows/test_optimum_common.yml)